### PR TITLE
Removed List name from List

### PR DESCRIPTION
**Pull request by Leslie303**
Removed list names from beatles list in place 0, and stones list in place 0. 
Updated code to convert to dictionaries so the loops iterate from value 0 in the two lists. 
Fixed a typo typeles to tuples.

### CODE REVIEW

File: BCE_3_2.py [hunk 1]
```diff
@@ -12,19 +12,19 @@
...
+beatles = [george, john, paul, ringo]
 #Print to verify output
 print("Task 1: Beatles list\n", beatles, "\n")
-#Output: ['beatles', ('George', 'guitar'), ('John', 'guitar'), ('Paul', 'bass'), ('Ringo', 'drums')]
+#Output: [('George', 'guitar'), ('John', 'guitar'), ('Paul', 'bass'), ('Ringo', 'drums')]
 
 #Task 2 Answer
 
 #Create dictionary
 beatles_dict = {}
```
Comment (billyean) [inline BCE_3_2.py]: Remove the key, you can easily accomplish this dictionary with one line code(same as stones_dict) 

```beatles_dict = {x[0]: x[1] for x in beatles}```

**Comment by billyean** [comment]
Also you may have seen there is conflict in your file, which means someone(Stephen) has changed the code after you check out the code.

File: BCE_3_2.py (hunk 1 above)
Comment (Leslie303) [reply BCE_3_2.py]: you can also do it by 

beatle_dict = dict(beatles) 

once the list name is out of the list.

**Comment by Leslie303** [comment]
@billyean , Yes, I think the issue is two changes to Line 19. I'll review what Stephen did since I check it out last night.

**Comment by billyean** [comment]
No need, I solved the conflict in the web, that's what people usually do

**Comment by Leslie303** [comment]
Thank you. I was uncertain about how to do that.

**Pull request merged**